### ISSUE TITLE
How do I add a product to an invoice

### ISSUE THREAD

**Issue by nickbe** [cytech/BillingTrack#28]
I feel stupid but I can't find how to do this - only how to add a lookup.

**Comment by cytech** [comment]
yeah, right now it only searches the item lookups so the product has to be in item lookups...
which only happens automatically if "Push active Products to Item Lookups" is set to yes in system settings.
